src/python/pants/backend/python/util_rules/BUILD

```diff
@@ -1,7 +1,11 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_sources()
+python_sources(
+    overrides={
+        "local_dists_pep660.py": dict(dependencies=["./scripts"]),
```

Comment:
should this dependency be specifically for the pep660 wrapper script perhaps, in cases there'll be more scripts in there in the future?

Reply:
Good point. I'll update this.